setup-azad-auth/beta.env

```diff
@@ -0,0 +1,22 @@
+# The hostname to update
+#host=universityofoxford.test.instructure.com
+host=universityofoxford.beta.instructure.com
+#host=universityofoxford.instructure.com
+
+# Bearer token
+token="12201~TEF1MhBoJMQrYxi2EbUdQAdCMCfNhOJ7du6ICwkc9o5nnIUiQbwKMdzPKQj5Yriu"
```

Comment:
Is this token supposed to be in github? PRs I think are publicly viewable things

Reply:
The rest looks fine to me